server/scsynth/SC_CoreAudio.h

```diff
@@ -246,6 +249,8 @@ class SC_CoreAudioDriver : public SC_AudioDriver {
                                         AudioBufferList* outOutputData, const AudioTimeStamp* inOutputTime,
                                         void* defptr);
 
+    const bool isClippingEnabled() { return mSafetyClipThreshold > 0 && mSafetyClipThreshold < INFINITY; }
```

Comment:
`const` on the return type doesn't have any effect, but `const` on the function indicates that the function doesn't mutate the object; i think that's what you intended:

```suggestion
    bool isClippingEnabled() const { return mSafetyClipThreshold > 0 && mSafetyClipThreshold < INFINITY; }
```